optimizely/config_manager.py

```diff
@@ -133,6 +135,7 @@ def _set_config(self, datafile):
             return
 
         self._config = config
+        self._configReadyEvent.set()
```

Comment:
What is the effect of calling `set()` twice

Reply:
No effect. It will make the already 'true' internal flag to 'true' again. Once an event has been set, all the threads which wait for the events do not block until .clear() has been called on the event, which makes the internal flag false.